### ISSUE TITLE
Feature Request. Reading CSV file as definitions

### ISSUE THREAD

**Issue by jmedzen** [dominiclet/obsidian-note-definitions#25]
I like this plugin's idea. One more easy way is to locate a CSV/txt file as a custom dictionary.

I been using a few specialized dictionary file to support my research all day. Which contains more than 50000 defines, if I convert each of them into separate files, which caused the obsidian searching and tagging view very slow, sometimes I get beachball rolling on my Macbook.
 
So if a single CSV/txt file would serve as a personal dictionary, I think It will be more efficient and easier to maintain.

Another suggest, can we customize the root definition folder name?

Thanks for the effort and sharing.

**Comment by chicman** [comment]
It is safe to say that enable support for multiple popular formats for large single file definition is always a plus. 
CSV and json are all well known formats and I believe it is easy to find parser component. CSV is just a txt file with comma ',' as data separator.

**Comment by dominiclet** [comment]
Yup I understand, but it will not be a priority at the moment. I will however look into the performance issue for large files that you mentioned and update this thread when there's progress.
